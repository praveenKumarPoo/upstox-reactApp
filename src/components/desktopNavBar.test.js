import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { BrowserRouter, Route } from "react-router-dom";
import DesktopNavBar from './desktopNavBar';
import renderer from "react-test-renderer";

afterEach(cleanup);
test('renders NavLink without crash need  to Router ', () => {
  render(<BrowserRouter><Route><DesktopNavBar /></Route></BrowserRouter>);
});

  test('Match Snapshort', () => {
    const tree = renderer.create(<BrowserRouter><Route><DesktopNavBar /></Route></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot()
  });

  