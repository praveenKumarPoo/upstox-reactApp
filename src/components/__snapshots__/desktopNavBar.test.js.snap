// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match Snapshort 1`] = `
<nav
  className="sc-AxjAm ffuoIy"
>
  <div
    className="logo"
  >
    UpStoX
  </div>
  <ul
    className="nav-list"
  >
    <li>
      <a
        className="link"
        data-testid="dashboard"
        href="/Dashboard"
        onClick={[Function]}
      >
        Dashboard
      </a>
    </li>
    <li>
      <a
        className="link"
        data-testid="LiveChart"
        href="/LiveChart"
        onClick={[Function]}
      >
        Live stock chart
      </a>
    </li>
  </ul>
  <button
    className="sc-AxirZ jLFzrT"
  >
    <img
      alt="open button"
      src="leftarrow.svg"
    />
  </button>
</nav>
`;
